@navikt/core/react/src/overlay/dismiss/DismissableLayer.tsx

```diff
@@ -329,20 +329,38 @@ const DismissableLayerNode = forwardRef<HTMLDivElement, DismissableLayerProps>(
      * we want to disable pointer events on the body when the first layer is opened.
      */
     useEffect(() => {
-      if (!node || !disableOutsidePointerEvents || index !== 0 || !enabled) {
-        return;
+      if (!node || !enabled) return;
+      if (disableOutsidePointerEvents) {
+        if (!changedBodyPointerEvents) {
```

Comment:
Can these ifs be merged?
```suggestion
      if (disableOutsidePointerEvents && !changedBodyPointerEvents) {
```